Rakefile

```
@@ -220,11 +220,11 @@ BASE_BRANCH_MAJOR_VERSION = if BASE_BRANCH == 'master'
                               Integer(BASE_BRANCH[/^\d+/])
                             end
 
-def create_pull_request(project_name, branch, base=BASE_BRANCH)
+def create_pull_request(project_name, branch, custom_pr_comment, base=BASE_BRANCH)
   github_client.create_pull_request(
     "rspec/#{project_name}", base, branch,
     "Updates from rspec-dev (#{Date.today.iso8601})",
-    "These are some updates, generated from rspec-dev's rake tasks."
+    "These are some updates, generated from rspec-dev's rake tasks.\n\n#{custom_pr_comment}"
```

Should probably check that its present and add the `\n\n` if so:
```suggestion
    ["These are some updates, generated from rspec-dev's rake tasks.", custom_pr_comment].join("\n\n")
```